BlueEastCode/bluerain-plugin-bootstrap
# Feature/checkbox

## What I did
Added Checkbox component
Added stories
Added Documentation
Added Test

## How to test
Use test file to test in test folder at root

Is this testable with jest or storyshots?
Yes. with Both.

Are any of the existing tests are failing ?
In this plugin, Tests are not running. Hence, not sure.

Does this need an update to the documentation?
No
Does it have breaking changes ?
No
If your answer is yes to any of these, please make sure to include it in your PR.


@bjavaid @amnajunaid please review the changes and merge them.
